crates/walrus-sdk/src/config.rs

```diff
@@ -32,11 +32,13 @@ mod committees_refresh_config;
 mod communication_config;
 mod reqwest_config;
 mod sliver_write_extra_time;
+mod upload_mode;
 
 pub use self::{
     committees_refresh_config::CommitteesRefreshConfig,
     communication_config::{ClientCommunicationConfig, CommunicationLimits},
     reqwest_config::RequestRateConfig,
+    upload_mode::UploadMode,
```

Comment:
Question/Suggestion: Could we also have a force parameter here?

Reply:
I strongly feel that "force" is an execution-time knob. Persisting it risks surprising overrides and complicates precedence imo.